internal/agent/hooks/hook.go

```diff
@@ -32,7 +32,11 @@ var FirstBoot = []Interface{
 }
 
 // AfterUkiInstall sets which Hooks to run after uki runs the install action
-var AfterUkiInstall = []Interface{}
+var AfterUkiInstall = []Interface{
+	&BundlePostInstall{},
+	&CustomMounts{},
+	&Lifecycle{},
```

Comment:
What about `&CopyLogs{}` ?

Reply:
In uki, the logs are copied as part of the encryption jhook as to not keeping closing and opening the encrypted devices several times.